src/app/components/EmbedConsentBanner/ConsentBanner.styles.tsx

```diff
@@ -16,6 +16,16 @@ export default {
       }`,
     }),
 
+  parentLive: ({ palette }: Theme) =>
+    css({
+      backgroundColor: palette.WHITE,
+      padding: `${pixelsToRem(16)}rem`,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'flex-start',
```

Comment:
Is it worth just setting `border: none` here if the only difference is to exclude border styling for Live pages?

Reply:
Ahh so the component will end up something like?
```jsx
<div
      data-testid="consentBanner"
      id={`consentBanner${id ? `-${id}` : ''}`}
      css={isLive ? consentBannerCss.parent + consentBannerCss.parentLive : consentBannerCss.parent + consentBannerCss.parentLive }
      ref={viewRef}
    >
```

Reply:
Yea with the `css` prop you can supply an array if you want to add multiple stylesheets, so something like this may work:
```
css={[consentBannerCss.parent, isLive && consentBannerCss.parentLive]}
```

Reply:
The only issue that _may_ happen is the priority of the styles getting applied. So the border from the `parent` style may overwrite the no border style from `parentLive`. Not 100% sure how Emotion prioritises these without using `!important`, which we would want to avoid.

Reply:
Seems to work, on Chrome anyway. I guess it depends how browsers interpret priority for what to use when rendering, chrome seems to take the last defined property, it's probably in the css spec so should be fine
![image](https://github.com/bbc/simorgh/assets/4706195/07866950-ed4a-4889-ba69-bd87069d2902)

Reply:
Yeah looks like last declaration always wins 
![image](https://github.com/bbc/simorgh/assets/4706195/8156d371-45b6-4d62-b175-04814e14d612)

Reply:
Ah nice, thanks for checking 💪